docs/sidebars.js

```diff
@@ -45,7 +45,6 @@ module.exports = {
       "cli/deploy-a-program",
       "offline-signing",
       "offline-signing/durable-nonce",
-      "cli/usage",
```

Comment:
why was this removed?

Reply:
This must have fallen through the cracks while merging...

Reply:
Re-added it.